### PR TITLE
config.yml updates + Application Info page

### PR DESCRIPTION
**Pull request by emiliom**
Application Info page:
- upper-section text
- updates to a few of the FAQs

Also updated _config.yml and _InstructionSiteUpdates.md

### CODE REVIEW

File: _pages/01-applicants.md [hunk 1]
```diff
@@ -1,23 +1,25 @@
 ---
 layout: applicant
 title: Information for Applicants
-description: <b>Due to COVID-19, there will be no in-person event in 2020. 
-    A virtual event around August is being discussed.
-    Applications will open soon.
-    Please check back in early May.</b><br><br>
+description: <b>Due to COVID-19, OceanHackWeek (OHW) 2020 will take place as a virtual, online event.
```
Comment (ocefpaf) [inline _pages/01-applicants.md:4]: What was the verdict OceanHackWeek or OceanHackweek?

File: _pages/01-applicants.md [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
+
+**To apply, please [fill out this form](https://form.jotform.com/201774238098159) by July 13, 2020.**  
+Accepted applicants will be notified no later than July 22, 2020.  
+We expect to accommodate approximately 50 participants.
 
-In Oceanhackweek we will explore the intersection of data science and oceanography through tutorials and hands-on “hacking” projects. In tutorials, we will learn data science tools, cloud computing, visualization, and a suite of software assets to interact with data sets of complex temporal-spatial structures or high volume. In project sessions, we will immediately put these skills to use by implementing research, computation, or visualization ideas in a group setting. To best benefit from the program, participant are expected to have some experience with Python programming and data analysis.
+**Formal daily activities will take place over a period of up to 3 hours per day, tentatively 10am-1pm US PDT / 1-4pm US EDT / 17:00-20:00 UTC.** Formal daily activities includes tutorials and project check-in with the entire group, but excludes actual project time. The time period will be finalized based on the geographical distribution of participants.
```
Comment (ocefpaf) [inline _pages/01-applicants.md:20]: "The time period will be finalized based on the geographical distribution of participants."

This is vague and may discourage people from applying and/or we may accept someone from a time-zone that won't be able to make it later.

I'd rather have a time slot that we know the instructors will be able to teach. CarpentryCon set proxies to repeat the record presentations for other time-zones. We don't have that but we can try to publish the recorded lesson ASAP and encourage people to watch them and still participate in the other parts of the event that are asynchronous, like the project and help desks.

File: _pages/01-applicants.md [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
+
+For additional questions, please email <a href="mailto:oceanhkw@uw.edu" target="_blank">oceanhkw@uw.edu</a>.
```
Comment (ocefpaf) [inline _pages/01-applicants.md:22]: Who answer this e-mail?

**Comment by ocefpaf** [comment]
Let's give some time for at least one more review before we merge this.

File: _pages/01-applicants.md (hunk 1 above)
Comment (emiliom) [reply _pages/01-applicants.md:4]: OceanHackWeek (it's what settled on for the NSF proposal)

File: _pages/01-applicants.md (hunk 3 above)
Comment (emiliom) [reply _pages/01-applicants.md:22]: Good question. For now I think only @leewujung and Jane (UW eScience) receive them. But we should talk about this and expand it soon.

File: _pages/01-applicants.md (hunk 2 above)
Comment (emiliom) [reply _pages/01-applicants.md:20]: I think you're suggesting that we can just drop the last sentence, right? I'm good with that, so I'll do it right away before merging.

Comment (emiliom) [reply _pages/01-applicants.md:20]: I've also removed the sentence from the application form text.

**Comment by emiliom** [comment]
Just waiting for the CI to complete, before merging ...

File: _pages/01-applicants.md (hunk 2 above)
Comment (ocefpaf) [reply _pages/01-applicants.md]: > I think you're suggesting that we can just drop the last sentence, right?

Yep.

**Pull request merged**